__tests__/ExpensiMark-HTML-test.js

```diff
@@ -896,7 +896,7 @@ test('Test for user mention without space or supported styling character', () =>
 
 test('Test for user mention with user email includes underscores', () => {
     const testString = '@_concierge_@expensify.com';
-    const resultString = '<mention-user>@_concierge_@expensify.com</mention-user>';
+    const resultString = '@_<a href=\"mailto:concierge_@expensify.com\">concierge_@expensify.com</a>';
```

Comment:
Is it possible the first `_` should be part of the email?
<img width="855" alt="image" src="https://github.com/Expensify/expensify-common/assets/165133/bc77cbe7-f002-4338-9706-c0957bb843e9">